bin/rollem.js

```diff
@@ -6,7 +6,7 @@ const fs = require('fs')
 const join = require('path').join
 const argv = require('minimist')(process.argv.slice(2))
 
-const configFilename = argv.c || 'rollem.config.js'
+const configFilename = (argv.c === true) ? 'rollem.config.js' : argv.c;
```

Comment:
@snuggs Ok, not a problem - I will remove the semicolon. But I still a "fan" of semicolons at the end of lines :)